usermods/AHT10_v2/usermod_aht10.h

```diff
@@ -0,0 +1,320 @@
+#pragma once
+
+#include "wled.h"
+#include <AHT10.h>
+
+#define AHT10_SUCCESS 1
+
+#define STATE_SETTING_ENABLED 0b00000001
+#define STATE_SETTING_MQTT_PUBLISH 0b00000010
+#define STATE_SETTING_MQTT_PUBLISHALWAYS 0b00000100
+#define STATE_SETTING_MQTT_HOMEASSISTANT 0b00001000
+#define STATE_INITDONE 0b00010000
+
+#define STATE_ISSET(stateFlags, state) (((stateFlags) & (state)) == (state))
```

Comment:
Why on Earth would you want the ancient method of bitfield handling via macros? What's wrong with unnamed bitfield struct?

```c++
struct {
  bool enabled : 1;
  bool mqtt : 1;
  ...
}
...
if (enabled) {
  ...
  if (mqtt) {
    ...
  }
}
```
This code is much easier to follow.

Reply:
I'm from a C# world - C++ is .. not my forté :)

Reply:
Fair. Look at the code for Segment class for example.